mods/cnc/languages/campaign/en.ftl

```diff
@@ -11,6 +11,11 @@ options-difficulty =
 ## campaign-maprules.yaml
 actor-moneycrate-name = Money Crate
 
+actor-airstrike-proxy =
+   .name = Air Strike
+   .description = Deploy an aerial napalm strike.
+    Burns buildings and infantry along a line.
+
```

Comment:
it seems this was added 3 times, in campaign, in regular rules, and in map twist of fate. As regular translations are present by default the others can be deleted

Reply:
Dedupcliated.